client/landing/stepper/declarative-flow/internals/steps-repository/design-setup/unified-design-picker.tsx

```diff
@@ -140,7 +140,7 @@ const UnifiedDesignPickerStep: Step = ( { navigation, flow } ) => {
 			intent,
 			seed: siteSlugOrId || undefined,
 			_locale: locale,
-			include_pattern_virtual_designs: isEnabled( 'virtual-themes/onboarding' ),
+			include_pattern_virtual_designs: true,
```

Comment:
Since we no longer need this param, shouldn't we remove it from the API endpoint and discard it entirely from here? :)

Reply:
Hmm... Good point!